Yoast/javascript
# 13996 send list of blocks to web worker

## Summary


This PR can be summarized in the following changelog entry:

* Adds a new attribute to the `Paper`: `wpBlocks`, to be used to send WordPress block editor data to the analysis.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* See test instructions on WordPress SEO free: https://github.com/Yoast/wordpress-seo/pull/14029

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://github.com/Yoast/wordpress-seo/issues/13996
